packages/component/src/Styles/StyleSet/Bubble.js

```diff
@@ -1,33 +1,156 @@
+/* eslint no-magic-numbers: ["error", { "ignore": [-1, 0, 1, 2, 10] }] */
+
+import Color from 'color';
+
+function acuteNubSVG(nubSize = 10, backgroundColor, color, strokeWidth = 1, side = 'bot', upSideDown = false) {
+  const halfNubSize = nubSize / 2;
+  const halfStrokeWidth = strokeWidth / 2;
+  const horizontalTransform =
+    side === 'bot' ? '' : `translate(${halfNubSize} 0) scale(-1 1) translate(${-halfNubSize} 0)`;
+  const verticalTransform = upSideDown ? `translate(0 ${halfNubSize}) scale(1 -1) translate(0 ${-halfNubSize})` : '';
+
+  const p1 = [nubSize, halfStrokeWidth].join(' ');
+  const p2 = [strokeWidth, halfStrokeWidth].join(' ');
+  const p3 = [nubSize + strokeWidth, nubSize + halfStrokeWidth].join(' ');
+
+  return `<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 ${nubSize} ${nubSize}"><g transform="${horizontalTransform} ${verticalTransform}"><path d="M${p1} L${p2} L${p3}" fill="${Color(
```

Comment:
remove xmlns? Are you sure we don't want this svg as a separate file?

Reply:
1. We should keep `xmlns`
1. Can't make it a separate file because we generate the content on-the-fly (different SVG for different nub sizes)